infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/AbstractWorkspaceServiceAccount.java

```diff
@@ -133,6 +134,15 @@ private void createImplicitRolesWithBindings(Client k8sClient) {
         singletonList("metrics.k8s.io"),
         Arrays.asList("list", "get", "watch"),
         serviceAccountName + "-metrics");
+
+    // secrets role
+    createRoleWithBinding(
```

Comment:
note that it won't work for DevWorkspaces.
Please contribute https://github.com/devfile/devworkspace-operator/blob/main/controllers/workspace/provision/rbac.go#L43 or create an issue for us https://github.com/devfile/devworkspace-operator/issues/new

Reply:
I've created an issue for that: https://github.com/devfile/devworkspace-operator/issues/483